Source/PlasticSourceControl/Private/PlasticSourceControlState.cpp

```diff
@@ -158,7 +160,7 @@ FName FPlasticSourceControlState::GetSmallIconName() const
 	{
 		return FName("Perforce.NotAtHeadRevision_Small");
 	}
-	else if (WorkspaceState != EWorkspaceState::CheckedOut)
+	else if (WorkspaceState != EWorkspaceState::CheckedOutChanged && (WorkspaceState != EWorkspaceState::CheckedOutUnchanged)
```

Comment:
Isn't there a compilation error there? Dangling `(` in the condition.

I guess it got missed by the compiler because of one of those (hateful) `#if-#endif` conditions.

`#if ENGINE_MAJOR_VERSION == 4`